phoenicis-apps/src/main/java/com/playonlinux/apps/dto/ApplicationDTO.java

```diff
@@ -70,14 +71,18 @@ public String getDescription() {
         return scripts;
     }
 
+    public static Comparator<ApplicationDTO> nameComparator() {
```

Comment:
Hum they are not exactly the same. If I want to merge them, I need to create something like a NameableDTO interface. Not sure it worth it